docs/4.0/getting-started/download.md

```diff
@@ -68,13 +68,13 @@ Bootstrap's `package.json` contains some additional metadata under the following
 Install Bootstrap in your Ruby apps using [Bundler](https://bundler.io/) (**recommended**) and [RubyGems](https://rubygems.org/) by adding the following line to your [`Gemfile`](https://bundler.io/gemfile.html):
 
 {% highlight ruby %}
-gem 'bootstrap', '~> 4.0.0.beta'
+gem 'bootstrap', '~> {{ site.current_version }}'
 {% endhighlight %}
 
 Alternatively, if you're not using Bundler, you can install the gem by running this command:
 
 {% highlight sh %}
-gem install bootstrap -v 4.0.0.beta
+gem install bootstrap -v {{ site.current_version }}
```

Comment:
The format of the rubygem version is slightly different:

```
site.current_version: 4.0.0-beta.2
     rubygem version: 4.0.0.beta2
```

`-` replaced by `.`, and no `.` between "beta" and "2".

Reply:
Hmm, @mdo, I'd say move the rubygem to a new _config.yml variable; this at least should make it easier to change it.

Reply:
Or alternatively, @glebm could change the tag starting from beta 2 and on?

Reply:
The rubygem version format is correct for rubygems and is not compatible with the tag here.

We could transform it with a Jekyll helper like this (or ERB code in _config.yml if Jekyll supports that):

```ruby
version.tr('-', '.').sub(/([a-z])\.(\d+)$/, '\1\2')
```

Reply:
well crap, fixing

Reply:
just pushed a new var for that

Reply:
OK, whatever I tried didn't work even with a custom Jekyll filter to pass a regex to gsub.

So I guess, the separate variable solution should do the job; we just need to document it so that we don't miss it.